plugins/nanospell/plugin.js

```diff
@@ -429,23 +416,36 @@
 				triggerSpelling((spellFastAfterSpacebar && (ch8r === CHARCODES.SPACE || ch8r === CHARCODES.LF || ch8r === CHARCODES.CR)), target)
 			}
 
+			function findNearestParentBlock(element) {
+				var elementPath = new CKEDITOR.dom.elementPath(element),
+					elements = elementPath.elements,
+					i;
+
+				for (i in elements) {
```

Comment:
Yep, you're right.  I was basing this off of a loop in one of the functions in the original plugin, but now I realize that order was not important in that case.  There's another one of these somewhere in here.  I'll fix them both